docs/sources/mimir/_index.md

```diff
@@ -12,18 +15,49 @@ keywords:
   - observability
 menuTitle: Grafana Mimir
 title: Grafana Mimir documentation
-weight: 1
+hero:
+  title: Grafana Mimir
+  level: 1
+  image: /media/docs/mimir/GrafanaLogo_Mimir_icon.png
+  width: 100
+  height: 100
+  description: Grafana Mimir is an open source software project that provides horizontally scalable, highly available, multi-tenant, long-term storage for Prometheus.
+cards:
+  title_class: pt-0 lh-1s
+  items:
+    - title: Get started
+      href: ./get-started/
+      description: Get started quickly with Grafana Mimir's extensive documentation, tutorials, and deployment tooling. Use the monolithic mode to get up and running with just one binary and no added dependencies.
+      height: 24
+    - title: Set up and configure
+      href: ./configure/about-configurations/
+      description: Set up Grafana Mimir with Helm, Puppet, or Jsonnet and Tanka. Migrate to Grafana Mimir from Cortex, Thanos, or Prometheus. Configure Grafana Mimir through the use of a YAML-based configuration file or CLI flags and learn about the common configurations.
+      height: 24
+    - title: Send metric data
+      href: ./send/
+      description: Configure your data sources to write data to Grafana Mimir. These include such sources as Prometheus, the OpenTelemetry Collector, and Grafana Agent.
```

Comment:
note: we'll have to rewrite our docs for Alloy from Agent and replace the old configurations with flow if I understand correctly

Reply:
> note: we'll have to rewrite our docs for Alloy from Agent and replace the old configurations with flow if I understand correctly

@krajorama  Understand and agree.  We have a task (the first for Taylor) underway to change the verbiage from Agent to Alloy.  The other database writers have been doing the same for their areas too.

Reply:
Yep, can confirm I'm working on this now, per https://github.com/grafana/alloy/issues/885.